packages/contracts/foundry.toml

```diff
@@ -13,6 +13,7 @@ extra_output = [
 optimizer = true
 optimizer_runs = 20_000
 force = false
+no_match_path = "^(?!.*\\/utils\\/)(?!.*utils).*$"
```

Comment:
There are only js scripts in the [utils](https://github.com/ubiquity/ubiquity-dollar/tree/development/utils) folder so `forge coverage` doesn't pick up js files for tests, only `.sol` files. I think you are free to delete this `no_match_path` setting